samples/config/dev-benchmark/README.md

```diff
@@ -0,0 +1,11 @@
+# Dev-benchmark sample config
+This config is used to develop and test gatling benchmark simulation.
```

Comment:
Gatling

In general, FR writing style is to capitalize product names (Gatling, Helm not gatling, helm) when referring to them in text *except* when you're referring to case-sensitive commands (i.e., Run the helm command, not run the Helm command).

Reply:
OK. Thanks for looking on it! Will try to remember this writing style.